spec/models/news_spec.rb

```diff
@@ -0,0 +1,60 @@
+# == Schema Information
+#
+# Table name: news
+#
+#  id         :integer          not null, primary key
+#  author     :string
+#  title      :string
+#  desc       :text
+#  source     :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+require 'rails_helper'
+
+RSpec.describe News, type: :model do
+
+  context "attributes" do
+    it 'has author' do
+      expect(build(:news, author: 'John Doe')).to have_attributes(author: 'John Doe')
+    end
+
+    it 'has title' do
+      expect(build(:news, title: 'Hello')).to have_attributes(title: 'Hello')
+    end
+
+    it 'has description' do
+      expect(build(:news, desc: 'Sample description')).to have_attributes(desc: 'Sample description')
+    end
+
+    it 'has source' do
+      expect(build(:news, source: 'http://google.com')).to have_attributes(source: 'http://google.com')
+    end
+  end
+
+  context "validations" do
```

Comment:
INFO: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)